mimic/model/clb_objects.py

```diff
@@ -289,6 +305,28 @@ def get_nodes(self, lb_id, node_id):
 
         return not_found_response("loadbalancer"), 404
 
+    def get_node_feed(self, lb_id, node_id):
+        """
+        Return load balancer's node's atom feed
+        """
+        if lb_id not in self.lbs:
+            return not_found_xml("Load balancer"), 404
+
+        _verify_and_update_lb_state(self, lb_id, False,
+                                    self.clock.seconds())
```

Comment:
I don't like this function at all. It pretends to be a private function in module scope of `mimic.canned_responses.loadbalancer`. It's actually used as a private ***method*** of `RegionalCLBCollection`. I'll let it slide because it's already wrong, but I'm also filing an issue.